lms/djangoapps/ccx/views.py

```diff
@@ -301,6 +303,14 @@ def create_ccx(request, course, ccx=None):
             user=request.user,
         )
 
+    # .. event_implemented_name: COURSE_CREATED
+    COURSE_CREATED.send_event(
+        time=datetime.now(timezone.utc),
```

Comment:
Could you please specify the argument?. @JuanDavidBuitrago 
```suggestion
        time=datetime.now(tz=timezone.utc),
```